about/source_code.markdown

```diff
@@ -13,29 +13,29 @@ MoveIt! code is hosted on github in the [ros-planning organization](http://githu
 
   * [moveit](https://github.com/ros-planning/moveit) - Main repo of MoveIt!, contains the following packages:
     * moveit - Metapackage
-    * moveit_core - Core C++ libraries
-    * moveit_ros - Metapackage
-    * moveit_ros_benchmarks
-    * moveit_ros_benchmarks_gui
-    * moveit_ros_manipulation - High level pick and place pipeline
-    * moveit_ros_move_group - Central ROS node for MoveIt!
+    * moveit_core - Core functionality including RobotModel, RobotState, collision checking
+    * moveit_ros_planning - planning components, execution manager, plugin loaders
+    * moveit_ros_move_group - The ``move_group`` main node for using MoveIt! via ROS messages
+    * moveit_ros_planning_interface - Python and ROS msg interfaces to communicate with ``move_group``
     * moveit_ros_perception - Octomap and other perception plugins
-    * moveit_ros_planning - Core ROS components and IK solvers
-    * moveit_ros_planning_interface - Python and ROS interfaces
+    * moveit_ros_manipulation - High level pick and place pipeline
     * moveit_ros_robot_interaction - Interative marker tools for Rviz
     * moveit_ros_visualization - Rviz tools
-    * moveit_ros_warehouse - Database connection
-    * moveit_planners - Metapackage
+    * moveit_ros_warehouse - Database plugins for storing scene and configuration data
+    * moveit_ros_benchmarks - Benchmarking using PlannerArena
+    * moveit_ros - Deprecated metapackage
+    * moveit_planners - Deprecated metapackage
```

Comment:
Why do you consider these metapackages to be deprecated?
Our main `moveit` metapackage relies on them and I see no reason why we should remove them at any point.